downstream/modules/platform/proc-central-auth-install-keycloak.adoc

```diff
@@ -0,0 +1,191 @@
+[id="proc-central-auth-install-keycloak"]
+
+= Install Keycloak with Ansible
+
+When you have installed the collection and dependencies necessary for your network to run the installation of Keycloak performs the following actions on  the system running it:
+
+* Creates user and group accounts
+
+* Downloads the installation archive from the Keycloak website
+
+* Unarchives the content you downloaded while ensuring that the files are associated with the correct users and privileges
+
+* Verifies that your version of Java Virtual Machine is up-to-date
+
+* Integrates Keycloak into the host service management system (e.g., systemd daemon)
+
+[NOTE]
+====
+Both user and group account names are keycloak by default.
+====
+
+. Create the following playbook:
++
+- name: Playbook for Keycloak Hosts
```

Comment:
I assume the playbook here should be rendering as a code block? I would take note of this to fix later.